config/initializers/highlight_override.rb

```diff
@@ -0,0 +1,54 @@
+# Monkey patch the generated Bindings for the swagger Highlight model
+
+Api::V0::Bindings::NewHighlight.class_exec do
+  def valid_location_strategies?
+    location_strategies.present? &&
+      location_strategies.none? { |item| item.nil? } &&
```

Comment:
`location_strategies.none?(&:nil?)` if you want to match the succinctness of the next line

Reply:
good call.   i think this is a good discussion point sometime.  both are idiomatic ruby, but one is more terse and potentially misunderstood by some dev?  it was just a little bit longer and more self evident which is why i used this form.  but both are fantastic.  will change to as you suggested to match the next line, which makes good sense.

Reply:
good call.   i think this is a good discussion point sometime.  both are idiomatic ruby, but one is more terse and potentially misunderstood by some dev?  it was just a little bit longer and more self evident which is why i used this form.  but both are fantastic.  will change to as you suggested to match the next line, which makes good sense.

Reply:
good call.   i think this is a good discussion point sometime.  both are idiomatic ruby, but one is more terse and potentially misunderstood by some dev?  it was just a little bit longer and more self evident which is why i used this form.  but both are fantastic.  will change to as you suggested to match the next line, which makes good sense.